### PR TITLE
Product changelog january

### PR DESCRIPTION
**Pull request by trichards-hiro**
## Description

A new January Release Roundup was recently released. Adding this content to the Docs site.

### CODE REVIEW

**Comment by LakshmiLavanyaKasturi** [comment]
Sidebar.js is missing from this PR.

File: docs/changelog-january.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: January 2023 Product Changelog
+---
+
+## Overview
+As Hiro works to provide the most complete and up-to-date information on product releases and updates, the format of the Product Changelog has been altered slightly to make the content easier to read and understand. Instead of deep dives into each individual product, the focus has shifted to the developer experience, and how these updates and changes improve the overall "developer journey."
```
Comment (LakshmiLavanyaKasturi) [inline docs/changelog-january.md:6]: each individual product -> each product

Comment (LakshmiLavanyaKasturi) [inline docs/changelog-january.md:6]: Comma can be removed after the developer experience.

File: docs/changelog-january.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+The sections below describe the changes and updates that were made in January 2023.
```
Comment (LakshmiLavanyaKasturi) [inline docs/changelog-january.md:8]: The sections below describe the changes and updates made in January 2023.

File: docs/changelog-january.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+## Developer onboarding
+The Hiro documentation team has been making significant progress in updating the technical documentation. The following updates/changes were added:
```
Comment (LakshmiLavanyaKasturi) [inline docs/changelog-january.md:11]: making significant progress updating the technical

File: docs/changelog-january.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+- a new Deployment Plans feature guide.
+- additional references for: Hiro Archive and Stacks Connect.
+- Hiro Contributors Guide that describes how to contribute to the technical documentation.
+
+## Empowering developers to do their best work
+Hiro wants to empower developers to do their best work with the latest updates and enhancements to the Hiro product offerings. The Stacks 2.1 upgrade, and its support across different products, include:
+
+- an API release candidate (v7.0.0) that supports Stacks 2.1, including a resolution for a known issue (genesis sync).
+
+**Note:** This API release candidate is currently in beta.
```
Comment (LakshmiLavanyaKasturi) [inline docs/changelog-january.md:23]: :::note
This API release candidate is currently in beta.
:::

File: docs/changelog-january.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+- Stacks 2.1 now supports Clarinet 1.4.1, Hiro Explorer, and Stacks.js.
+- Subnets has been improved with an updated Subnets contract compatible with Clarity2.
+- A new trait for NFTs and FTs has been added on the subnet, which specifies a required name and signature for registration. This update ensures the Stacks Blockchain API always knows the correct name for the mint function.
+
+## Additional bitcoin support across hiro products
+New API endpoints are currently being developed to support Bitcoin-related queries, and a Stacks.js client library to connect Bitcoin Core RPC. These improvements enable STX-to-BTC and BTC-to-STX address conversions.
```
Comment (LakshmiLavanyaKasturi) [inline docs/changelog-january.md:30]: enable to address conversations.

File: docs/changelog-january.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+- an API release candidate (v7.0.0) that supports Stacks 2.1, including a resolution for a known issue (genesis sync).
+
+**Note:** This API release candidate is currently in beta.
+
+- Stacks 2.1 now supports Clarinet 1.4.1, Hiro Explorer, and Stacks.js.
+- Subnets has been improved with an updated Subnets contract compatible with Clarity2.
+- A new trait for NFTs and FTs has been added on the subnet, which specifies a required name and signature for registration. This update ensures the Stacks Blockchain API always knows the correct name for the mint function.
+
+## Additional bitcoin support across hiro products
```
Comment (LakshmiLavanyaKasturi) [inline docs/changelog-january.md:29]: Hiro products

File: docs/changelog-january.md (hunk 5 above)
Comment (LakshmiLavanyaKasturi) [inline docs/changelog-january.md:30]: queries and a Stacks.js library....

File: docs/changelog-january.md (hunk 2 above)
Comment (trichards-hiro) [reply docs/changelog-january.md]: fixed

File: docs/changelog-january.md (hunk 3 above)
Comment (trichards-hiro) [reply docs/changelog-january.md]: fixed

File: docs/changelog-january.md (hunk 4 above)
Comment (trichards-hiro) [reply docs/changelog-january.md:23]: fixed

File: docs/changelog-january.md (hunk 5 above)
Comment (trichards-hiro) [reply docs/changelog-january.md]: fixed

File: docs/changelog-january.md (hunk 6 above)
Comment (trichards-hiro) [reply docs/changelog-january.md]: fixed

File: docs/changelog-january.md (hunk 5 above)
Comment (trichards-hiro) [reply docs/changelog-january.md]: fixed

File: docs/changelog-january.md (hunk 1 above)
Comment (trichards-hiro) [reply docs/changelog-january.md]: fixed

**Comment by trichards-hiro** [comment]
closing this PR to create a cleaner version without merge errors

**Pull request closed without merging**